### DISCUSSION TITLE
Bring back IRC channel, move to Libera.Chat

### DISCUSSION THREAD

**Discussion by JPcode05** [jeremyevans/autoforme#41] [General]
In [this commit](https://github.com/jeremyevans/autoforme/commit/9251d085dc500f755815873ffb5355c5adad42ea) you removed the link to your IRC channel on Freenode.

Since no one else talked about it, I would like the IRC channel `#forme` to be brought back on [Libera.Chat](https://libera.chat). Would you guys like to do that?

**Comment by jeremyevans** [comment]
When Freenode imploded, I decided to drop a supported IRC channel for all my libraries.  In recent years there was low usage, especially with AutoForme, so I don't think the benefits outweigh the maintenance cost.  If you have questions about AutoForme, it's probably best to ask them in GitHub Discussions.
